Code/Editor/EditorFramework/EditorApp/EditorApp.cpp

```diff
@@ -53,6 +57,9 @@ void ezQtEditorApp::SlotTimedUpdate()
   }
   ezTaskSystem::FinishFrameTasks();
 
+  // Close the splash screen when we get to the first idle event
+  CloseSplashScreen();
```

Comment:
Shouldn't it get closed when the first idle event arrives, not when we send it?

Reply:
The SlotTimedUpdate() function is connected to the idle timer event and thus is called when the message loop is processed.
IdleEvent is then just our relay of that information / timer event happening. So this seemed like a good place to have the quick function call.